docs/playbook/getting-set-up/notebook-setup.md

```diff
@@ -0,0 +1,196 @@
+
+-  **How to install Git**
+
+   - Install git by following this link - [Git Guides](https://github.com/git-guides/install-git?msclkid=af84852cb0f211ec9ad73e08c013d33c)
+
+-  **How to check your Git is working properly**
+   - Open terminal by searching for cmd or command prompt in Windows
+   - Type ```git version``` and press enter
+   - The output will tell you which version of Git you have installed
+   - If it says ```git is an unknown command``` then you don't have git installed
+
+
+**Docker Desktop**
+
+Docker is essentially a way of opening a virtual machine but for just one application.
+We use Docker Desktop just to open a Jupyter Server locally so we have an environment to code Python but with the environment that is very similar to the AWS Glue Environment
+
+- **Benefits of using Docker Desktop**
+
+  - Means we don’t need to have a separate computer which will run our Jupyter Environment
+  - Docker can be run on most operating systems
+    
+
+- **Installing Docker Desktop**
+
+  - Install [Docker Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows?msclkid=d06d5c4caeb011ec8b168b9f3d7e1e4e)
+  - You also need the Linux Kernel Package found [Here](https://docs.microsoft.com/en-gb/windows/wsl/install-manual#step-4---download-the-linux-kernel-update-package)
+
+**Data Platform Access**
+
+You need access to the Data Platform to run the code which will access the Data Platform's Data
+
+- **How to Check if your Data Platform Access is ok**
+
+  - Click this [link](https://hackney.awsapps.com/start#/) to access Single Sign-on for AWS Services
+  - Click `[AWS Account]` and check if you have any users with `DataPlatform` in its name
+  - If you can't do this, you need a username for the Data Platform
+
+## Notebook set up
+
+**Setting up the Local Files**
+1. Navigate to where you want the Data Platform Folder to be stored in `Windows Explorer`
+
+
+2. Open the Terminal in that area by entering `cmd` in the address bar in Windows Explorer
+   - You should see a terminal window with the path you navigated to `C:\Users\<your user name in windows>\<somewhere>`
+    
+
+3. Enter `git clone https://github.com/LBHackney-IT/Data-Platform.git` 
+   - This will download the entire Data Platform Repository onto your Local drive, where you navigated to
+
+
+4. Open the newly created `Data-Platform` folder and then Open the `notebook` folder
+
+
+5. Open the terminal in this folder like above.
+
+
+6. Enter `type nul > .env`. This creates an empty .env file in the notebook folder, if this worked properly, you should see a .env file in that folder
+
+
+7. Now open the `aws-config` folder
+
+
+8. Open the terminal in this folder like above.
+
+
+9. Enter `type nul > credentials`. 
+   - This creates an empty credentials file in the aws-config folder, if this worked properly, you should see a credentials file with no type. 
+
+**Alternate Set Up Method using Batch Files**
```

Comment:
Not sure we need to include this bit on batch files? One approach should be fine